validator/client/slashing_protection_interchange_test.go

```diff
@@ -1,168 +0,0 @@
-package client
```

Comment:
Why are we deleting this test?

Reply:
nice catch, super important we do not lose this test